### PR TITLE
Update readme alex

### LINKED ISSUE #121: Update Weekly openmeteo pipeline README
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

- Edit README so that it is up to date with weekly openmeteo pipeline progress

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #121 

I have done the following:
Updated the readme to include information on:
- Setup
- Usage
- How it works

### CODE REVIEW

File: weekly-openmeteo/README.md [hunk 1]
```diff
@@ -2,7 +2,87 @@
...
+5. Add to your `.env` file the following:
+    - REGION - region of your ECR repository
+    - ECR_URI - endpoint of the ECR repository
+    - IMAGE_NAME - a name given to your docker image
+
+6. Run the following command to automatically dockerise and push to ECR:
+```bash
+bash ../dashboard/deploy.sh
+```
+7. Now follow the terraform setup steps in the `../terraform` folder to provision a lambda based off this docker image
```
Comment (olliejth) [inline weekly-openmeteo/README.md:48]: Nice formatting of code sections.

File: weekly-openmeteo/README.md [hunk 2]
```diff
@@ -2,7 +2,87 @@
...
+python -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+3. Create a `.env` file with the following:
+    - DB_HOST - to access your RDS instance
+    - DB_USER - username for accessing RDS
+    - DB_PASSWORD - password for accessing RDS
+    - DB_NAME - the name of your database
+    - DB_PORT - port to access RDS (typically 5432 for postgres)
```
Comment (CalebMathers) [inline weekly-openmeteo/README.md:20]: Nice to explain what each env variable should contain, even with descriptive names

File: weekly-openmeteo/requirements.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 requests
-psycopg2
+psycopg2-binary
```
Comment (CalebMathers) [inline weekly-openmeteo/requirements.txt:2]: Better one to use, less problems

File: weekly-openmeteo/README.md [hunk 4]
```diff
@@ -2,7 +2,87 @@
...
+
+- To run tests run the following command:
+```bash
+pytest
+```
+Alternatively to run specific tests or test files:
+```bash
+pytest path/to/test_file.py
+pytest path/to/test_file.py::test_function_name
+```
```
Comment (CalebMathers) [inline weekly-openmeteo/README.md:64]: Good to have test commands in code blocks for easy copy pasting

File: weekly-openmeteo/README.md [hunk 5]
```diff
@@ -2,7 +2,87 @@
 This folder contains the code for the [OpenMeteo API](https://open-meteo.com/en/docs) data pipeline. This includes `extract.py`, `transform.py`, `load.py` and `pipeline.py`. This pipeline uses the `requests` library to send HTML GET requests to the `OpenMeteo API` and the `psycopg2` library to load this data onto our database. The data requested includes `time of sunset` and `time of sunrise`. This pipeline is therefore designed to run every week as these are far more predictable values.
 
 ## Setup
+1. Ensure that an SQL server RDS has been setup prior and is accessible
+    - Note: be sure to store the credentials for accessing this database safely.
+
+### To Run Locally:
```
Comment (CalebMathers) [inline weekly-openmeteo/README.md:8]: Nice clear instructions broken up into logical chunks

**Review by CalebMathers — APPROVED** [review]

**Review by olliejth — APPROVED** [review]
Well documented README.

File: weekly-openmeteo/requirements.txt (hunk 3 above)
Comment (MurasatoNaoya) [reply weekly-openmeteo/requirements.txt:2]: Yes.

File: weekly-openmeteo/README.md [hunk 6]
```diff
@@ -2,7 +2,87 @@
...
+```
+To get a coverage report execute the following;
+```bash
+pytest --cov
+```
 
-## How it works
\ No newline at end of file
+## How it works
+#### `extract.py`
```
Comment (MurasatoNaoya) [inline weekly-openmeteo/README.md:71]: Clear high-level explanations of what exactly each script does and some of their dependencies.

**Review by MurasatoNaoya — COMMENTED** [review]
Clear, descriptive README. LGTM.

**Pull request merged**